src/sentry/features/temporary.py

```diff
@@ -514,6 +514,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:streamlined-publishing-flow", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     # Enable per-project selection for Jira integration
     manager.add("organizations:jira-per-project-statuses", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
+    # Enable using paginated projects endpoint for Jira integration
+    manager.add("organizations:jira-paginated-projects", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
```

Comment:
Maybe lingering from a previous commit, but I'm confused why there are no merge conflicts 👀 

Wasn't this already included in https://github.com/getsentry/sentry/pull/89709/?

Reply:
o yeah it was, this shouldn't be here :oopsie